src/plotman/plotters/madmax.py

```diff
@@ -649,3 +665,131 @@ def _cli_974d6e5f1440f68c48492122ca33828a98864dfc() -> None:
 )
 def _cli_aaa3214d4abbd49bb99c2ec087e27c765424cd65() -> None:
     pass
+
+
+# Madmax Git on 2022-03-26 -> https://github.com/madMAx43v3r/chia-plotter/commit/ecec17d25cd547fa4bb64b2eb7455b831c8a2882
+@commands.register(version=(3,))
+@click.command()
+# https://github.com/madMAx43v3r/chia-plotter/blob/ecec17d25cd547fa4bb64b2eb7455b831c8a2882/LICENSE
+# https://github.com/madMAx43v3r/chia-plotter/blob/ecec17d25cd547fa4bb64b2eb7455b831c8a2882/src/chia_plot.cpp#L257-L277
+@click.option(
+    "-k",
+    "--size",
+    help="K size (default = 32, k <= 34)",
+    type=int,
+    default=32,
+    show_default=True,
+)
+@click.option(
+    "-x",
+    "--port",
+    help="Network port (default = 8444, chives = 9699, mmx = 11337)",
+    type=int,
+    default=8444,
+    show_default=True,
+)
+@click.option(
+    "-n",
+    "--count",
+    help="Number of plots to create (default = 1, -1 = infinite)",
+    type=int,
+    default=1,
+    show_default=True,
+)
+@click.option(
+    "-r",
+    "--threads",
+    help="Number of threads (default = 4)",
+    type=int,
+    default=4,
+    show_default=True,
+)
+@click.option(
+    "-u",
+    "--buckets",
+    help="Number of buckets (default = 256)",
+    type=int,
+    default=256,
+    show_default=True,
+)
+@click.option(
+    "-v",
+    "--buckets3",
+    help="Number of buckets for phase 3+4 (default = buckets)",
+    type=int,
+    default=256,
+)
+@click.option(
+    "-t",
+    "--tmpdir",
+    help="Temporary directory, needs ~220 GiB (default = $PWD)",
```

Comment:
i assume that's for k32?

Reply:
Probably.  At the top of this list of options is a link to the code from which this was translated.  We don't expose this anywhere, I just decided to try to be complete so that this code is easier to think about.